systemd/fmn-worker@.service

```diff
@@ -1,14 +0,0 @@
-[Unit]
```

Comment:
I see you are taking this fmn-worker service away, but do you want to provide an example Celery service file in its place? If so, I suggest keeping the existing name.

Reply:
So I've added a replacement, although I do expect it to change as I fiddle with celery more.